cloudbuild/nightly/scripts/table_write.sh

```diff
@@ -36,13 +36,13 @@ timestamp=$(date +"%Y%m%d%H%M%S")
 JOB_ID=$(echo "$RANDOM" | md5sum | cut -c 1-30)
 # Adds timestamp to job_id to prevent repetition.
 JOB_ID="$JOB_ID"_"$timestamp"
-echo [LOGS: "$PROJECT_NAME"."$DATASET_NAME"."$SOURCE_TABLE_NAME" Write Test] Created JOB ID: "$JOB_ID"
 
 if [ "$MODE" == "bounded" ]
 then
   echo "Bounded Mode!"
+  echo [LOGS: "$PROJECT_NAME"."$DATASET_NAME"."$SOURCE" Write Test] Created JOB ID: "$JOB_ID"
```

Comment:
Let's modify to form one log stating testing for bounded mode and the subsequent message